dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -8,23 +8,25 @@
 TTS_BUCKET = 'cdo-tts'
 
 VOICES = {
-  # british child candidate
-  rosie: {
-    VOICE: 'rosie22k',
+  'en-us': {
+    VOICE: 'sharon22k',
     SPEED: 180,
     SHAPE: 100
   },
-  # american child candidate
-  ella: {
-    VOICE: 'ella22k',
-    SPEED: 140,
-    SHAPE: 98
+  'es-es': {
```

Comment:
Would it be appropriate to also add es-mx with voice #3? I want to add support for es-mx as soon as we fix the two letter language bug on Code Studio. And we might as well include it since we have approval/feedback from our partners, and I don't want to forget which one they chose. Then I can add it to ENABLED_LANGUAGES later?